src/accelerate/utils/memory.py

```diff
@@ -92,14 +92,15 @@ def release_memory(*objects):
 
 def should_reduce_batch_size(exception: Exception) -> bool:
     """
-    Checks if `exception` relates to CUDA out-of-memory, CUDNN not supported, or CPU out-of-memory
+    Checks if `exception` relates to CUDA out-of-memory, XPU out-of-memory, CUDNN not supported, or CPU out-of-memory
 
     Args:
         exception (`Exception`):
             An exception
     """
     _statements = [
         "CUDA out of memory.",  # CUDA OOM
+        "XPU out of memory.", # XPU OOM
```

Comment:
The linter complains:

```suggestion
        "XPU out of memory.",  # XPU OOM
```